src/playground/components/Configuration.js

```diff
@@ -105,10 +105,35 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
         value: ruleName,
         label: rulesMeta[ruleName].deprecated ? ruleName.concat(" (deprecated)") : ruleName
     }));
-    const [selectedRule, setSelectedRule] = useState();
+    const [selectedRules, setSelectedRules] = useState([]);
     const ruleInputRef = useRef(null);
     const [rulesWithInvalidConfigs, setRulesWithInvalidConfigs] = useState(new Set([]));
 
+    const handleRuleChange = () => {
+        selectedRules.forEach(selectedRule => {
+            if (ruleNames.includes(selectedRule)) {
+                options.rules[selectedRule] = ["error"];
+                onUpdate(Object.assign({}, options));
+                ruleInputRef.current.setValue("");
+            }
+        });
+    };
+
+    const Input = props => {
+        if (props.isHidden) {
+            return <components.Input {...props} />;
+        }
+        return (
+            <components.Input {...props} onKeyDown={e => {
+                if ((e.metaKey || e.ctrlKey) && e.code === "Enter") {
+                    handleRuleChange();
+                } else {
+                    props.onKeyDown(e);
+                }
```

Comment:
When we press `cmd + enter` when the dropdown is present it doesn't add all the rules but selects one rule from the dropdown and scrolls to one of the selected rules.

https://user-images.githubusercontent.com/46647141/204116153-999f2499-4d39-4dc9-a09d-f052c83910d9.mov

Reply:
weird... will check

Reply:
Same in windows also not working.